cmd/sonobuoy/app/operations/gen_plugin_test.go

```diff
@@ -0,0 +1,38 @@
+package operations
+
+import (
+	"testing"
+
+	"k8s.io/api/core/v1"
+	kuberuntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func TestLoadNonexistantPlugin(t *testing.T) {
+	_, err := GeneratePluginManifest(GenPluginConfig{
+		Paths:      []string{"./plugins.d"},
+		PluginName: "non-existant-plugin",
+	})
+
+	if err.Error() != "expected 1 plugin, got 0" {
+		t.Errorf("unexpected or no error %q", err)
+	}
+}
+
+func TestLoadRealPlugin(t *testing.T) {
+	bytes, err := GeneratePluginManifest(GenPluginConfig{
+		// Tests are executed with cwd set to their containing directory
+		Paths:      []string{"../../../../plugins.d"},
```

Comment:
Could you add a //TODO to fix this later.

Reply:
I don't know how we'd go about fixing it honestly

Reply:
Typically env-var for the root directory is passed around.